runtime/src/parachains.rs

```diff
@@ -224,6 +330,24 @@ fn localized_payload(statement: Statement, parent_hash: ::primitives::Hash) -> V
 }
 
 impl<T: Trait> Module<T> {
+	/// Dispatch some messages from a parachain, .
```

Comment:
Mind expanding it? Otherwise:

```suggestion
	/// Dispatch some messages from a parachain.
```